docs/backup-recovery/README.md

```diff
@@ -0,0 +1,150 @@
+# Cluster upgrades - backup and recovery #
+
+## Background ##
+
+This feature creates a pre-upgrade backup and provides a procedure for rapid recovery of a SNO in the event of a failed upgrade. In case of an upgrade failure, this feature allows the SNO to be restored to a working state with the previous version of OCP without requiring a re-provision of the application(s).  
+
+## Create Recovery Partition ##
+
+> **WARNING**: It is highly recommended to create a recovery partition at install time if opted to use this feature.
+
+There are two methods related to partition creation is described here:
+
+### Preferred method with Siteconfig ###
+
+The recovery partition can be created at install time by defining a node level property called `diskPartition` in the SiteConfig CR. The configuration for the partition must be defined by `start` parameter which must indicate, in mebibytes, the end of the `/sysroot` RHCOS partition and the start for the newly requested recovery partition.
+
+Additionally, a spare disk can be used for recovery partition as well and can be configured at provisioning time or at day 2 operation. A recovery partition of 50GB with mountpoint at `/var/recovery` can be created by following the below example. Notice that the deviceName can be referenced by its path, label or id as well, e.g. `/dev/disk/by-id/scsi-355cd2e404b7c23f3`
+
+```
+nodes:
+      - hostName: "snonode.sno-worker-0.e2e.bos.redhat.com"
+        role: "master"
+        rootDeviceHints:
+          hctl: "0:2:0:0"
+          deviceName: /dev/sda
+        ........
+        ........
+        #Disk /dev/sda: 893.3 GiB, 959119884288 bytes, 1873281024 sectors
+        diskPartition:
+          - device: /dev/sda
+            partitions:
+              - mount_point: /var/recovery
+                size: 51200
+                start: 800000
+
+```
+
+### Alternative method with Extra-manifest
+
+The alternative way a recovery partition can be created at install time by defining an extra-manifest MachineConfig, as described here:<br>
```

Comment:
I'd either remove 'The alternative way' or just rephrase this paragraph as: "As an alternative way, a recovery partition can also be created at install time"

Reply:
Rephrased